packages/runtime/src/widgets/Stepper/Stepper.tsx

```diff
@@ -127,7 +127,13 @@ const Stepper: React.FC<StepperProps> = (props) => {
   }
 
   const stepPercentage = 100 / stepsLength;
-  const activeStepPercentage = (activeStep + 1) * stepPercentage;
+  const activeStepPercentage = activeStep
+    ? (activeStep + 1) * stepPercentage
+    : 0;
+
+  if (activeStep === undefined) {
+    return null;
+  }
```

Comment:
This can be moved before the calculations above to save on some perf/additional undefined checking